.github/.OwlBot.lock.yaml

```diff
@@ -13,5 +13,5 @@
 # limitations under the License.
 docker:
   image: gcr.io/cloud-devrel-public-resources/owlbot-java:latest
-  digest: sha256:894d56f45fc3f4f0d5f3bcf790846419ee2d8e44715eae8917d6a1bba2b7283d
-  # created: 2022-10-26T00:00:00.000000000Z
+  digest: sha256:edae91ccdd2dded2f572ec341a768ad180305a3e8fbfd93064b28e237d35920a
```

Comment:
Copying from the latest owl-bot-copy pull request:

https://github.com/googleapis/java-accessapproval/pull/879/files#diff-28faeab193819b6210664df33e757a79c3b066449bc2b5cce3ebe463ec1e4f3fR16